docs/working-remotely-ssh.md

```diff
@@ -0,0 +1,49 @@
+# Working remotely with Visual Studio Code
+
+It is possible to work remotely on a machine containing the required opensafely software using Visual Studio Code.
+This may be useful in cases where it is not possible to install all of the required sofware to run opensafely on your local machine, or where running this required software severly affects the performance of your local machine. 
+
+This guide walks you through the steps required to set up the machine running the opensafely software (the "server") and the local machine from which you will access it (the "client").
+
+## 1. Server setup
+
+First, ensure that the opensafely software and all of its pre-requisites have been installed using the [Getting Started guide](getting-started). If using this guide to connect to Gitpod, please skip to step 2.
+
+### SSH server installation and configuration
+The steps in this guide require enabling access to the server using secure shell protocol (SSH).
+On macOS, this requires enabling the [Remote Login](https://support.apple.com/en-gb/guide/mac-help/mchlp1066/mac) feature. 
+On Windows, this requires installation of the [OpenSSH Server optional feature](https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse).
+On Linux, this requires installation of the OpenSSH Server such as is described in these [instructions for Ubuntu Linux](https://ubuntu.com/server/docs/service-openssh).
+
```

Comment:
If we're having to explain at a basic level how to configure SSH, maybe we should add a mention that it is better to authenticate with a public/private key, over a password, and to set a passphrase for that private key.

* I'm not entirely sure if the default for Windows is to allow username/password access which isn't very good; see this [key management page from Microsoft](https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_keymanagement for how to do so). From a quick look, I also think the macOS default is to allow password authentication.
* The linked Ubuntu guide does explain how to generate a key pair. However, it doesn't make the importance of setting a passphrase for a private key clear. Those instructions state "During the process you will be prompted for a password. Simply hit *Enter* when prompted to create the key."